sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample06_Transactions.md

```diff
@@ -44,6 +44,48 @@ using (var ts = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled))
 }
 ```
 
+### Transactions across entities
+In the previous version of the library, when attempting to perform transactions that spanned multiple entities, it was necessary to configure your senders so that they would send-via the other entity in the transaction. With updates to the service, the new library introduces the concept of transaction groups to handle cross entity transactions. When creating senders and receivers that should be part of a cross-entity transaction, you can specify the same `TransactionGroup` string in the options as shown below. 
```

Comment:
Is there a sample around `send-via` or documentation on it that we can link?   I'm not familiar with the semantics and implications, and it would have been helpful to read about, since it's assumed knowledge in this sample.

Reply:
I think part of the issue here is that this new feature is meant to abstract away the whole send-via concept, but it doesn't do so completely. I'm debating whether it is even worth re-introducing this concept to an audience that may not be aware of it vs just listing the perhaps arbitrary seeming rules that are a result of the underlying implementation.